python/gtsam/utils/plot.py

```diff
@@ -36,18 +36,15 @@ def set_axes_equal(fignum):
     ax.set_zlim3d([origin[2] - radius, origin[2] + radius])
 
 
-def ellipsoid(xc, yc, zc, rx, ry, rz, n):
+def ellipsoid(rx, ry, rz, n):
```

Comment:
API change? Why?

Reply:
The center point values are not used in the function anywhere. They are simply wasteful args that serve no purpose.